core/src/main/scala/scalaz/std/Future.scala

```diff
@@ -5,7 +5,8 @@ import scala.concurrent.{Await, CanAwait, ExecutionContext, Future}
 import scala.concurrent.duration.Duration
 
 trait FutureInstances1 {
-  implicit def futureInstance(implicit ec: ExecutionContext): Monad[Future] with Cobind[Future] =
+
+  implicit def futureInstance(implicit ec: ExecutionContext): Cobind[Future] with MonadError[({type λ[α,β] = Future[β]})#λ, Throwable] with Catchable[Future] =
```

Comment:
@vmarquez Eh. Well, how about, give your signature a new name, and take off the `implicit` keyword from `futureInstance` and deprecate it.  It's a simpler case than #582; no new priority layers are needed, your instance should go where the current one does.  That will still yield a mima warning, but might be except-able.